tests/util/common.ts

```diff
@@ -0,0 +1,6 @@
+// Sort dict by key
+export function sortObjectByKeys(o) {
```

Comment:
You can probably inline this code, sorting object keys is a very _niche_ operation that's not commonly used. Personally for comparison I convert the object to an array.

```typescript
const expected = Object.keys(items).sort().map((key) => ({ key, value: items[key] });
```

Alternatively, `expect` / `chaijs` can deep compare objects via

```typescript
expect({ a: 1, b: 2 }).to.deep.include({ b: 2 });
```